packages/neon/neon_talk/lib/src/widgets/message.dart

```diff
@@ -465,19 +465,45 @@ class _TalkCommentMessageState extends State<TalkCommentMessage> {
         final separateMessages = widget.chatMessage.actorId != widget.previousChatMessage?.actorId ||
             widget.previousChatMessage?.messageType == spreed.MessageType.system ||
             previousDate == null ||
-            date.difference(previousDate) > const Duration(minutes: 3);
+            date.difference(previousDate) > const Duration(minutes: 3) ||
+            widget.chatMessage.lastEditTimestamp != null;
 
-        Widget? displayName;
+        Widget? label;
         Widget? avatar;
         Widget? time;
         if (separateMessages) {
-          displayName = Text(
+          label = Text(
             getActorDisplayName(TalkLocalizations.of(context), widget.chatMessage),
             style: textTheme.labelLarge!.copyWith(
               color: labelColor,
             ),
           );
 
+          if (widget.chatMessage.lastEditTimestamp != null && widget.chatMessage.lastEditActorDisplayName != null) {
+            final timestamp = DateTimeUtils.fromSecondsSinceEpoch(
+              tz.local,
+              widget.chatMessage.lastEditTimestamp!,
+            );
+
+            label = Row(
+              children: [
+                label,
+                Tooltip(
+                  message: TalkLocalizations.of(context).roomMessageLastEdited(
+                    widget.chatMessage.lastEditActorDisplayName!,
+                    DateFormat.yMd().add_jm().format(timestamp),
+                  ),
+                  child: Text(
+                    ' (${TalkLocalizations.of(context).roomMessageEdited})',
```

Comment:
why isn't the `( )` part of the localized string?

Reply:
Because they will always be present, so they don't need to be translated?